incubator/nodejs-express/image/project/Dockerfile

```diff
@@ -8,15 +8,16 @@ RUN apt-get update \
  && echo 'Finished installing dependencies'
 
 # Install stack dependencies
+WORKDIR /project/node_modules
```

Comment:
The intent was to always create the node_modules folder as if there are no dependencies then it might not get created and then the tar commands could fail. But in this case, this is not really an issue as we know exactly what is in the stack. 

Deleting this line.